docs/reference/cli.mdx

```diff
@@ -0,0 +1,130 @@
+The Nitric CLI allows you to build, develop, and deploy your serverless application.
+
+To verify your installation run `nitric version`. The output should look something like this:
+
+```
+Go Version: go1.17.8
+Go OS/Arch: darwin/amd64
+Git commit: fce97c9
+Build time: 2022-03-23T14:05:59+1100
+Nitric CLI: v1.1.0-develop.14-98-gfce97c91c5bce6eeb864194e592134322b52d8e5
+```
+
+To get a list of the available commands, run the following command:
+
+```
+nitric help
+```
+
+## Common Commands
+
+There are a few common commands that you will use frequently:
+
+- `nitric new` creates a new project
+- `nitric stack new` creates a new stack within a project
+- `nitric run` runs your stack locally
+- `nitric up` deploys your stack to the cloud
+- `nitric down` destroys your stack from the cloud
+
+## Environment Variables
+
+### Pulumi Access Token
+
+To view your deployments on the pulumi dashboard, you will need a pulumi access token. You can get this token by logging into pulumi on the browser and going to your profile settings. Under the 'Access Tokens' tab click 'Create token'.
+
+Add the environment variable to your `~/.zshrc` or `~/.bashrc` as:
+
```

Comment:
Might want to move this into an FAQ - don't want to confuse the user into thinking that they need to configure this or use the pulumi dashboard as it is not required for the general developer.

Reply:
Amended.